AxonDeepSeg/download_model.py

```diff
@@ -36,6 +30,14 @@ def download_model(destination = None):
             "ERROR: Data was not succesfully downloaded and unzipped- please check your link and filename and try again."
         )
 
+    files_after = listdir()
+
+    # retrieving unknown model folder names
+    model_folders = list(set(files_after)-set(files_before))
```

Comment:
This is the simplest way I found to retrieve the folder names automatically after the download.
Ideally, if the actual download wasn't processed by the `download_data` utility, retrieving the folder names would be trivial.

Reply:
I would personally use **Pathlib's** `iterdir()` function than **os** module` listdir() `function. Although both of have the same functionality, the reason for having preferance to `pathlib` module is that ADS uses `pathlib` to handle paths ( see [here](https://github.com/neuropoly/axondeepseg/wiki/How-to-handle-paths)) and it will maintain consistency in the codebase.

Reply:
Thanks for the link! I was looking for that exact document yesterday